### ISSUE TITLE
.rubocop.yml is not respected

### ISSUE THREAD

**Issue by LeEnno** [SublimeLinter/SublimeLinter-haml-lint#4]
This is my `.rubocop.yml`, located in `/Users/Enno/Sites/pictrs/master/pictrs/.rubocop.yml`:

``` yml
Style/HashSyntax:
  Enabled: false
```

This is my haml file:

``` haml
%div{ :id => 'container' }
```

This is the console output:

```
/Users/Enno/Sites/pictrs/master/pictrs/.rubocop.yml
SublimeLinter: hamllint: test.haml ['/Users/Enno/.rvm/bin/rvm-auto-ruby', '-S', 'haml-lint'] 
SublimeLinter: hamllint output:
/var/folders/l2/j37t2cjs5ssgcgsrk1xyt1x40000gn/T/SublimeLinter3-Enno/test.haml:1 [W] RuboCop: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
```

[SublimeLinter-rubocop](https://github.com/SublimeLinter/SublimeLinter-rubocop) detecs the config file just fine.

Any ideas?

**Comment by jeroenj** [comment]
Is it detected correctly when linting without this SublimeLinter plugin? If so the problem should be in the plugin. I am no longer using (nor using Sublime Text) so it's a bit hard to debug this for me. But I'd be happy to accept a merge request if you find a fix for it.

**Comment by LeEnno** [comment]
Unfortunately I have no clue of python, but maybe someone else will help.

**Comment by jeroenj** [comment]
Discussed in #3 and described in #5.

TL;DR `haml_lint` (not `haml-lint`) v0.13.0 or higher is required.
